doc/modules/ensemble.rst

```diff
@@ -601,65 +611,118 @@ chapter on gradient boosting in [F2001]_ and is related to the parameter
 Mathematical formulation
 -------------------------
 
-GBRT considers additive models of the following form:
+We first present GBRT for regression, and then detail the classification
+case.
+
+Regression
+^^^^^^^^^^
+
+GBRT regressors are additive models whose prediction :math:`y_i` for a
+given input :math:`x_i` is of the following form:
 
   .. math::
 
-    F(x) = \sum_{m=1}^{M} \gamma_m h_m(x)
+    \hat{y_i} = F_M(x_i) = \sum_{m=1}^{M} h_m(x_i)
 
-where :math:`h_m(x)` are the basis functions which are usually called
-*weak learners* in the context of boosting. Gradient Tree Boosting
-uses :ref:`decision trees <tree>` of fixed size as weak
-learners. Decision trees have a number of abilities that make them
-valuable for boosting, namely the ability to handle data of mixed type
-and the ability to model complex functions.
+where the :math:`h_m` are estimators called *weak learners* in the context
+of boosting. Gradient Tree Boosting uses :ref:`decision tree regressors
+<tree>` of fixed size as weak learners. The constant M corresponds to the
+`n_estimators` parameter.
 
-Similar to other boosting algorithms, GBRT builds the additive model in
-a greedy fashion:
+Similar to other boosting algorithms, a GBRT is built in a greedy fashion:
 
   .. math::
 
-    F_m(x) = F_{m-1}(x) + \gamma_m h_m(x),
```

Comment:
What is the reason for removing gamma?

Reply:
Do you include gamma in h?

Reply:
the gamma thing has been replaced by the note about updating the leaves values (it's actually what it's used for).

Also it was incorrect: gamma should be a function of the samples, not a single value as it was before